flocker/node/agents/blockdevice.py

```diff
@@ -725,6 +741,28 @@ def detach_volume(self, blockdevice_id):
         )
         volume_path.moveTo(new_path)
 
+    def resize_volume(self, blockdevice_id, size):
+        """
+        Change the size of the loopback backing file.
+
+        Sparseness is maintained by using ``truncate`` on the backing file.
+
+        This implementation is limited to being able to resize volumes only if
+        they are unattached.
+        """
+        backing_path = self._unattached_directory.child(
+            blockdevice_id.encode("ascii")
+        )
+        try:
+            backing_file = backing_path.open("r+")
+        except IOError:
+            raise UnknownVolume(blockdevice_id)
+        else:
+            try:
+                backing_file.truncate(size)
+            finally:
+                backing_file.close()
+
```

Comment:
Docstring for interface claims this returns something. Either this or the interface should be changed so that they match.